### DISCUSSION TITLE
Routines

### DISCUSSION THREAD

**Discussion by lwychan** [nielsmaerten/gluco-check#88] [Q&A] [answered]
Hi @nielsmaerten , 

I'm unclear on how to setup the routines so different querys to google assistant will produce my blood glucose?
I have input "Check my blood sugars," "Check my bloods" & "Check my sugars" but I'm not quite sure how to get them to link up with the task of talking to nightscout. Could you provide any further information please?

**Comment by nielsmaerten** [comment, marked as answer]
Hi @lwychan  
[This video might help](https://glucocheck.app/assets/routines-setup.mp4). It shows the entire process of creating a new Routine.  

⚠ Important!  
In the video, we're linking _"Check my sugar"_ to the command _"Talk to Gluco Check"_.  
If you're using Nightscout Status, you'll have to replace that with **"Talk to Nightscout Status"**.  

Gluco Check is still in beta, so _"Talk to Gluco Check"_ won't work yet.
(But if you'd like to try Gluco Check, message us at beta@glucocheck.app and we'll get you signed up :)

**Reply by lwychan** [reply]
Awesome, that worked a treat, I did do that on my own, but I think I did question marks which is a bit silly, but changed that and it started working, keep up the good work, really helping out 👍🏻

**Reply by lwychan** [reply]
@nielsmaerten Is there any integration with Google home hub, nest hub? Like is there a tile or background that is your nightscout chart?

**Reply by nielsmaerten** [reply]
There's currently nothing like that planned, no. Like the idea though.
I don't have the time, or a Hub at the moment. But if some dev is reading this and feels inspired, let me know ;)

**Reply by lwychan** [reply]
Time for me to learn the art of development ☺️
